examples/advanced.html

```diff
@@ -33,12 +33,26 @@ <h1>Media Chrome Advanced Video Usage Example</h1>
         <media-time-range></media-time-range>
         <media-time-display show-duration remaining></media-time-display>
         <media-captions-button></media-captions-button>
+        <media-loading-indicator></media-loading-indicator>
         <media-playback-rate-button></media-playback-rate-button>
         <media-pip-button></media-pip-button>
         <media-fullscreen-button></media-fullscreen-button>
       </media-control-bar>
     </media-controller>
     <div class="examples">
+      <button id="loadingBtn">Toggle Loading</button>
```

Comment:
we should update this example to not rely on the javascript snippet and instead only rely on the built-in behavior + CSS. (this was added for testing/iteration as I was implementing, but wasn't expected to be the reference example).